packages/components/pie-button/src/defs.ts

```diff
@@ -1,30 +1,27 @@
 /**
  * Button size variants
  */
-export enum BUTTON_SIZE {
-    XSMALL = 'xsmall',
-    SMALL_EXPRESSIVE = 'small-expressive',
-    SMALL_PRODUCTIVE = 'small-productive',
-    MEDIUM = 'medium',
-    LARGE = 'large'
-}
+export type BUTTON_SIZE =
+    | 'xsmall'
+    | 'small-expressive'
+    | 'small-productive'
+    | 'medium'
+    | 'large';
 
 /**
  * Button style variants
  */
-export enum BUTTON_TYPE {
-    SUBMIT = 'submit',
-    BUTTON = 'button',
-    RESET = 'reset',
-    MENU = 'menu'
-}
+export type BUTTON_TYPE =
```

Comment:
Perhaps follow Skip's example and export these in uppercase like `ButtonType`, `ButtonSize` etc